WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/UpdateEventsAPIV5.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class UpdateEventsAPIV5: UpdateEventsAPIV4 {
+
+    override var apiVersion: APIVersion {
+        .v5
+    }
+
+    private var basePath: String {
+        "/notifications"
+    }
+
+    override func getLastUpdateEvent(selfClientID: String) async throws -> UpdateEventEnvelope {
+        var components = URLComponents(string: "\(pathPrefix)\(basePath)/last")
+        components?.queryItems = [URLQueryItem(name: "client", value: selfClientID)]
+
+        guard let path = components?.string else {
+            throw UpdateEventsAPIError.invalidPath
+        }
+
+        let request = HTTPRequest(
+            path: path,
+            method: .get
+        )
+
+        let response = try await httpClient.executeRequest(request)
+
+        // Change: 400 error removed.
```

Comment:
Nitpick:
if this is the only change, one could also make a comment in the implementation in V0 like:
"can be removed when minimum version becomes v5"...
But this way it's a bit more explicit, so both ways are fine for me.

Reply:
I'm not sure I completely understand. What can be removed? The error?

Reply:
Ah I see, hmm I think I prefer to be more explicit so there's no need to figure out some subtle api differences later on then previous versions are removed.